series/procedures/validate_cutter_range.py

```diff
@@ -0,0 +1,34 @@
+from book_code_generation.location_number_creation import generate_author_number, number_between
+
+
+class InvalidCutterRangeError(Exception):
+    def __init__(self, message):
+        # Call the base class constructor with the parameters it needs
+        super().__init__(message)
+        self.message = message
+
+
+def validate_cutter_range(series, custom_name, letter, number):
```

Comment:
TODO: switch from 'series' as input to 'location' as input; move procedure to book_code_generation package.